.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   release:
     runs-on: ubuntu-latest
-    if: github.repository_owner == 'voxpupuli'
+    if: github.repository_owner == 'puppetlabs'
     steps:
       - uses: actions/checkout@v4
       - name: Install Ruby 3.0
```

Comment:
Down below on line 32: `Publish gem to GitHub packages` I think we need to excise that from this release workflow?

Reply:
Tim requested that we continue to publish to GitHub packages: https://github.com/puppetlabs/beaker-puppet/pull/242#discussion_r1484745134

Though I'm not sure if things will still Just Work with our current secrets?

Reply:
Can't wait to find out!